### PR TITLE
Removing Cleanup Webhook Configurations from the website doc

### PR DESCRIPTION
**Pull request by NoSkillGirl**
related PR - https://github.com/kyverno/kyverno/pull/2263

### CODE REVIEW

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -456,16 +456,4 @@ kubectl delete -f https://raw.githubusercontent.com/kyverno/kyverno/main/definit
 
 ```sh
 helm uninstall kyverno --namespace kyverno kyverno/kyverno
-```
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:459]: These set of backticks need to remain.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Leave line 459 intact for the Helm uninstall command code block.

File: content/en/docs/Installation/_index.md (hunk 1 above)
Comment (NoSkillGirl) [reply content/en/docs/Installation/_index.md:459]: I haven't removed the backtics. this git diff is kind of misleading. 

check the backticks [here](https://github.com/kyverno/website/pull/253/commits/87d93a1c574ba558b6117119c5c5a222a3b8d831#diff-2304e00d07492b5b6f7e83ed1a4f4050f0ebbbee5d5a2fc3cd4dd30bfe1cfc78R459)

**Comment by NoSkillGirl** [comment]
I haven't removed the backtics. this git diff is kind of misleading.

check the backticks here

**Comment by chipzoller** [comment]
I see it was added at the bottom. This is fine then.

**Review by chipzoller — APPROVED** [review]

**Comment by NoSkillGirl** [comment]
@chipzoller, @JimBugwadia - I do not have permission to merge this PR. Can one of you please merge it?

**Pull request merged**